infrastructure/ssz/src/main/java/tech/pegasys/teku/infrastructure/ssz/schema/SszCollectionSchema.java

```diff
@@ -49,7 +49,6 @@ default SszCollectionT createFromElements(final List<? extends SszElementT> elem
   }
 
   default TreeNode createTreeFromElements(final List<? extends SszElementT> elements) {
-    // TODO: probably suboptimal method implementation:
```

Comment:
I'd leave the comment of this being prob a suboptimal impl

Reply:
Problem is how we will revisit it if the comment is gone. Possibly raise an issue?

Reply:
Literally 80% of our code is in the same spot....

Reply:
also please note the entire comment of the reasoning is there if you look at the next lines. its just the 'probably suboptimal' that is gone.
added more noise with https://github.com/Consensys/teku/issues/9035 to also track onthe board but i think where i left it was more than adequate. am marking this thread resolved.